test/test_nestedtensor.py

```diff
@@ -235,11 +235,7 @@ def test_is_contiguous(self):
 
         # Test non_contiguous case
         assert not nt_noncontiguous.is_contiguous()
-        self.assertRaisesRegex(
-            RuntimeError,
-            r"clone_nested only supports memory format Preserve, but got Contiguous instead.",
-            lambda: nt_noncontiguous.contiguous()
-        )
+        self.assertEqual(nt_contiguous, nt_noncontiguous.contiguous())
```

Comment:
huh, this PR fixed `.contiguous()` for nested tensors?

Reply:
oh nevermind - just re-read the PR description :)

Reply:
yeah because contiguous calls `clone` under the hood. And previously we did not support MemoryFormat::Contiguous but his PR introduced the change